src/templates/Plugin.js

```diff
@@ -74,43 +97,41 @@ const Body = ({
       )}
 
       {!isEmpty(installHtml) && (
-        <>
-          <div className="mb-4" id="installation-instructions">
-            <Title className="text-3xl">Installation Instructions</Title>
+        <div className="mb-24">
+          <div className="mb-4" id={PAGE_SECTIONS.INSTALLATION_INSTRUCTIONS.fragment}>
+            <Title className="text-3xl">{PAGE_SECTIONS.INSTALLATION_INSTRUCTIONS.label}</Title>
           </div>
 
           <div className="mb-4">
             <RoadieDocsLink availableOnRoadie={availableOnRoadie} roadieDocsPath={roadieDocsPath} />
           </div>
 
-          <div className="mb-10">
-            <div
-              className="mb-4 mt-0 prose prose-primary max-w-none"
-              dangerouslySetInnerHTML={{
-                __html: installationInstructions.childMarkdownRemark.html,
-              }}
-            />
-          </div>
-        </>
+          <div
+            className="mt-0 prose prose-primary max-w-none"
+            dangerouslySetInnerHTML={{
+              __html: installationInstructions.childMarkdownRemark.html,
+            }}
+          />
+        </div>
       )}
 
       {!isEmpty(notesHtml) && (
-        <>
-          <div className="mb-4" id="notes">
-            <Title className="text-3xl">Things to Know</Title>
+        <div className="mb-24">
+          <div className="mb-4" id={PAGE_SECTIONS.NOTES.fragment}>
+            <Title className="text-3xl">{PAGE_SECTIONS.NOTES.label}</Title>
           </div>
 
           <div
-            className="mb-10 prose prose-primary max-w-none"
+            className="prose prose-primary max-w-none"
             dangerouslySetInnerHTML={{ __html: notes.childMarkdownRemark.html }}
           />
-        </>
+        </div>
       )}
 
       {!isEmpty(changelog) && (
-        <>
-          <div className="mb-4" id="changelog">
-            <Title className="text-3xl">Changelog</Title>
+        <div className="mb-24">
```

Comment:
Increased the whitespace between the sections to make them appear more distinct.